src/scheduler/limits.c

```diff
@@ -1423,11 +1420,10 @@ check_server_max_group_res(server_info *si, queue_info *qi, resource_resv *rr,
 		case 0:
 			return (0);
 		case 1:	/* generic group limit exceeded */
-			schderr_args_server_res(NULL, resname, err);
 			err->rdef = rdef;
 			return (SERVER_GROUP_RES_LIMIT_REACHED);
 		case 2:	/* individual group limit exceeded */
-			schderr_args_server_res(rr->group, resname, err);
+			schderr_args_server_res(rr->group, rdef->name, err);
```

Comment:
Why copy the rdef name into a string buffer instead of just using it directly in translate_fail_code().  The function already uses rdef directly.

Reply:
Well I thought that resname was kind of redundant now that we have resource definition. I'm just passing it for completeness of the structure here, especially because group will still be part of error structure

Reply:
If you feel strongly about it then let me know and I'll change it